uaclient/entitlements/tests/test_livepatch.py

```diff
@@ -361,90 +408,210 @@ def test_can_enable_true_on_entitlement_inactive(
-            ("5.0.0-00-generic", True),
+            (
+                system.KernelInfo(
+                    uname_release="3.5.0-00-generic",
+                    proc_version_signature_full="",
+                    proc_version_signature_version="",
+                    version="3.5.0",
+                    major=3,
+                    minor=5,
+                    patch=0,
+                    abi="00",
+                    subrev="",
+                    hwerev="",
+                    flavor="generic",
+                ),
+                False,
+            ),
+            (
+                system.KernelInfo(
+                    uname_release="4.3.0-00-generic",
+                    proc_version_signature_full="",
+                    proc_version_signature_version="",
+                    version="4.3.0",
+                    major=4,
+                    minor=3,
+                    patch=0,
+                    abi="00",
+                    subrev="",
+                    hwerev="",
+                    flavor="generic",
+                ),
+                False,
+            ),
+            (
+                system.KernelInfo(
+                    uname_release="4.4.0-00-generic",
+                    proc_version_signature_full="",
+                    proc_version_signature_version="",
+                    version="4.4.0",
+                    major=4,
+                    minor=4,
+                    patch=0,
+                    abi="00",
+                    subrev="",
+                    hwerev="",
+                    flavor="generic",
+                ),
+                True,
+            ),
+            (
+                system.KernelInfo(
+                    uname_release="4.10.0-00-generic",
+                    proc_version_signature_full="",
+                    proc_version_signature_version="",
+                    version="4.10.0",
+                    major=4,
+                    minor=10,
+                    patch=0,
+                    abi="00",
+                    subrev="",
+                    hwerev="",
+                    flavor="generic",
+                ),
+                True,
+            ),
+            (
+                system.KernelInfo(
+                    uname_release="5.0.0-00-generic",
+                    proc_version_signature_full="",
+                    proc_version_signature_version="",
+                    version="5.0.0",
+                    major=5,
+                    minor=0,
+                    patch=0,
+                    abi="00",
+                    subrev="",
+                    hwerev="",
+                    flavor="generic",
+                ),
+                True,
+            ),
         ),
     )
+    @mock.patch("uaclient.system.get_kernel_info")
+    @mock.patch(
+        "uaclient.system.get_platform_info",
+        return_value=PLATFORM_INFO_SUPPORTED,
+    )
     def test_can_enable_false_on_unsupported_min_kernel_version(
```

Comment:
Can't `test_can_enable_false_on_unsupported_kernel_min_version` be merged with this test? I don't see a difference between those

Reply:
Nice catch! I'll combine the cases.